distributed/tests/test_client.py

```diff
@@ -5595,9 +5596,9 @@ async def test_warn_when_submitting_large_values(c, s, a, b):
 
 @gen_cluster(client=True)
 async def test_unhashable_function(c, s, a, b):
-    d = {"a": 1}
-    result = await c.submit(d.get, "a")
-    assert result == 1
+    func = _UnhashableCallable()
+    result = await c.submit(func, 1)
```

Comment:
This change is because `dict.get` became hashable starting with Python 3.8. This is why `test_unhashable_function` was only failing in our Python 3.7 CI builds